lisa/node.py

```diff
@@ -1116,6 +1116,10 @@ def cleanup(self) -> None:
         for node in self._list:
             node.cleanup()
 
+    def clear(self) -> None:
```

Comment:
It doesn't need this method. If set env.status to new, the nodes will be reset too. Check the status methods of env.

Reply:
removed

Reply:
Is latest change pushed?

Reply:
missed this, updated